Backend_Development_With_PHP/Coding_Practices/Date_And_Time/DateandTime.md

```diff
@@ -0,0 +1,171 @@
+# PHP : Date and Time
+
+Dates are very important part of everyday life with the help of which we are able to track our work regularly without even thinking. Some of the most used operations in PHP while designing a website or SQL query execution are date and time. PHP provides powerful tools for date arithmetic that makes it easy to manipulate date. Some of the predefined functions in PHP for date and time are discussed below.
+
+## PHP date() Function   
+
+The PHP date() function is used to convert a timestamp into more readable date and time format. The computer stores dates and times in a format called UNIX Timestamp, which measures time as number of seconds since the beginning of the Unix epoch (midnight Greenwich Mean Time on January 1, 1970 i.e. January 1, 1970 00:00:00 GMT ). Since it is not possible for a human to resd or understand this, PHP converts a timestamp in a format which is more understandable by human. 
```

Comment:
would be better if we've date() and other functions under backticks like `date()` through \`date()\`